### ISSUE TITLE
[Arcs] Having Rogue Admirals and Attacking Another Regent 

### ISSUE THREAD

**Issue by Smokey-Chan** [LederCards/cards#57]
We know that a player with Rogue Admirals can attack the Empire alone, and when they do so, they can choose which Imperial ships attack and which defend. However, it is still unclear what happens when this player attacks another Regent in systems with Imperial ships. 

Can the player with Rogue Admirals still choose the attacking and defending ships

-OR-

MUST the Imperial ships join the defender?

**Comment by turmoilbyrd** [comment]
slightly related https://github.com/LederCards/cards/issues/38

**Comment by jdyearsley** [comment]
Yes, the player with Rogue Admirals will still pick and choose attacking and defending Imperial ships. I'll add an FAQ here.

**Comment by turmoilbyrd** [comment]
> Yes, the player with Rogue Admirals will still pick and choose attacking and defending Imperial ships. I'll add an FAQ here.

Ok, oh... I know balancing might not be the priority concern here but I feel this makes Rogue Admiral even stronger.

Currently **Rogue Admiral** can already move away imperial ships (1 action), or battle the Empire first and destroy most of the Imperial ships (1 action). So I don't feel they particularly need to be even stronger.

**Comment by jdyearsley** [comment]
Yeah, that's fair, but it's the most coherent way of doing it from a thematic and a rules-consistency angle.

**Comment by turmoilbyrd** [comment]
At least thematically, the way I justified it was: Imperial ships are nominally loyal to the Empire, but really they are more loyal to local regents in the Reach. So if you battle the Empire, some of them will break ranks. But if you attack a regent, they are bound by vows to protect the regents

**Comment by Gurnapster** [comment]
I agree with turmoilbyrd on this one. Since you can justify it thematically either way, it seems like determining the rules for purposes of balancing may be best

**Comment by turmoilbyrd** [comment]
Either way, a meta will develop around Rogue Admiral.

Now I feel Admiral is a bigger risk than asset for the Steward, who is less likely to want to help Admiral succeed in Act 1.

**Comment by jdyearsley** [comment]
From a rules angle, it's very difficult for me to justify a pick-and-choose case vs the Empire but not vs Regents. If anything, it makes more sense to allow a pick-and-choose for Regents but not the Empire. 

Here are the relevant rules:
> When a Regent attacks in battle, they may add Imperial ships in the battle system to their attacking ships.
> When a Regent defends in battle, they must add Imperial ships in the battle system to their defending ships.
> If the Empire is defending, the defending pieces are the Imperial ships in the battle system, and the First Regent takes any destroyed pieces of the Outlaw as Trophies. If there is no First Regent, they return to the Outlaw.

I would prefer one of these rulings:
1. Pick and choose in either case. (Emphasizing Regent Attacker rule.)
2. Pick and choose for Regent defender. Imperial ships defend for Empire defender. (Emphasizing Regent Attacker and Empire Defender rules.)

Rogue Admirals is meant to enable them to bring Imperial might to bear against other Regents. Thoughts?

**Comment by turmoilbyrd** [comment]
I see, so the Regent Defender rule is de-prioritized for the others

>  When a Regent defends in battle, they must add Imperial ships in the battle system to their defending ships.

From a purely rules angle they both seem reasonable.

Thematically, I can see (2) making sense. So in this version, the Imperial ships are in fact primarily loyal to the Empire. This does make sense to me.

**Comment by jdyearsley** [comment]
I'm fine with (2). This unfortunately is a ruling reversal for Rogue Admirals specifically, but overall I think this is more sensible and flows with the rules as written better.

**Comment by emptierset** [comment]
As a followup, does it require errata for a Rogue Admirals holding Regent to attack the Empire in the first place? Such a Regent ignores a rule that says "you cannot X" (where X is "choose Imperial ships as defender in battle"), but I don't believe there's a rule that says "Regents can X" in the first place. That is, generally speaking, ignoring "you are forbidden from X" does not necessarily allow you to X. Outlaws can battle the Empire because of this rule:

> Outlaws may choose either a Regent or the Empire as defender. If the Empire is defending, the defending pieces are the Imperial ships in the battle system.

But it feels more to me like the rule would need to look like this for Rogue Admirals to matter here:

> ~~Outlaws~~ Players may choose either a Regent or the Empire as defender. If the Empire is defending, the defending pieces are the Imperial ships in the battle system.

**Comment by turmoilbyrd** [comment]
I think the cleanest way to write it is probably

> Players may battle the Empire
> Truce Law: Regents cannot battle the Empire
